src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -521,7 +523,9 @@ const FileField = props => {
                 {description && <p>{description}</p>}
                 {!file.uploading && (
                   <>
-                    <strong id={fileNameId}>{file.name}</strong>
+                    <strong id={fileNameId} className="dd-privacy-mask">
+                      {file.name}
+                    </strong>
                     {file?.size && <div> {displayFileSize(file.size)}</div>}
```

Comment:
What do you think about hiding this as well?

Reply:
I think hiding is fine. I figured either would work here, but if you feel hiding is better, I can switch it.